app/windows/Welcome/Components/ToS.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+function ToS () {
```

Comment:
We should probably fetch this content from an URL to be always sure to have the latest content.
https://siderus.io/tos.html

Reply:
> Refused to display 'https://siderus.io/tos.html' in a frame because it set 'X-Frame-Options' to 'deny'

We need to allow CORS.

Reply:
Will a _download_ work? You don’t need to use an iFrame, but we could download the content and then display it. What do you say?

Reply:
Updated. We should keep in mind though that it relies on an internet connection, but I guess that shouldn't be a problem.

Reply:
Yes, it already relies on it due to the Siderus Network